calcom/cal.com
# fix: including parentId when filter by eventTypeId happens

## What does this PR do?

- When as admin/owner filters and select a eventTypeId that it's currently managed event type, count fails since using eventType filter will only look for specific id, now we add OR lookup for eventTypeId OR parentId.

Fixes #10261 

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Are there environment variables that should be set? No.
- What are the minimal test data to have? Having yarn seed-insights could help.
- What is expected (happy path) to have (input and output)? Managed event type bookings should be count when viewing team insights with a specific eventTypeId selected.
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
